src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -397,6 +380,25 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     `;
   }
 
+  private gainedFocus() {
+    const peopleList = this.renderRoot.querySelector('.people-list');
+    const peopleInput = this.renderRoot.querySelector('.people-chosen-input') as HTMLInputElement;
+    peopleInput.focus();
+    peopleInput.select();
+    if (peopleList) {
+      // Mouse is focused on input
+      peopleList.setAttribute('style', 'display:block');
+    }
+  }
+  private lostFocus() {
+    const peopleList = this.renderRoot.querySelector('.people-list');
+    if (peopleList) {
+      setTimeout(() => {
+        peopleList.setAttribute('style', 'display:none');
+      }, 300);
```

Comment:
Is this a delay for animation sake or to work-around an issue?

Reply:
This is a workaround, since technically when the user selects person via mouse click, onblur is activated (since the peopleList is outside of the search input). This would cause the list to 'display:none' but not select the user. This ensures the user is still selected.